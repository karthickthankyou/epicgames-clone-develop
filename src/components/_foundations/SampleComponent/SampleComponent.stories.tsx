import React from 'react'
import { ComponentStory, ComponentMeta } from '@storybook/react'
import SampleComponent from './SampleComponent'

export default {
  title: 'components/_foundations/SampleComponent',
  component: SampleComponent,
} as ComponentMeta<typeof SampleComponent>

const Template: ComponentStory<typeof SampleComponent> = () => (
  <SampleComponent />
)

export const Primary = Template.bind({})
Primary.args = {}
Primary.parameters = {}
