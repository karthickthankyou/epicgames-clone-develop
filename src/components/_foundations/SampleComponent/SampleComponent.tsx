export interface ISampleComponentProps {}

const SampleComponent = () => (
  <div>Hello, This is SampleComponent component! ðŸ›«</div>
)

export default SampleComponent
