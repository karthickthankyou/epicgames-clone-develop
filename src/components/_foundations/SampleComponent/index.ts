import SampleComponent from './SampleComponent'

export default SampleComponent
