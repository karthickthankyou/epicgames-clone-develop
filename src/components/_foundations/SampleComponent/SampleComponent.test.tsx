import { render } from '@testing-library/react'
import SampleComponent from './SampleComponent'

describe('SampleComponent Component', () => {
  test('it should match the snapshot', () => {
    const { asFragment } = render(<SampleComponent />)
    expect(asFragment()).toMatchSnapshot()
  })
})
