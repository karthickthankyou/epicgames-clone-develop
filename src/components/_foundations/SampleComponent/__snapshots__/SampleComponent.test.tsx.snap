// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleComponent Component it should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    ðŸ›¬  Hello, This is SampleComponent component! ðŸ›«
  </div>
</DocumentFragment>
`;
